transports/sql/native-delayed-delivery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: SQL Server Native Delayed Delivery
+summary: Describes the native delayed delivery implementation in the SQL Server transport
+reviewed: 2017-06-05
+component: SqlServer
+versions: '[3.1,)'
+---
+
+In Versions 3.1 and above, the SQL Server transport can take advantage of native [delayed delivery](/nservicebus/messaging/delayed-delivery.md) without then need to run the [timeout manager](/nservicebus/messaging/timeout-manager.md). Instead, the transport creates infrastructure which can delay messages using native SQL Server transport features. To enable the native delayed delivery use the following API:
+
+snippet: EnableNativeDelayedDelivery
+
+NOTE: In this mode the timeout manager will still be running in order to process all outstanding delayed messages. Refer to the [Disabling the timeout manager](/transports/sql/native-delayed-delivery.md#backwards-compatibility-disabling-the-timeout-manager) section for details on how to disable the timeout manager entirely.
+
+## How it works
+
+The transport creates an additional queue that stores delayed messages. The table name has the format `endpoint-name.suffix`, using the suffix specified in the configuration:
+
+snippet: DelayedDeliveryTableSuffix
+
+The timeouts poller checks for expired messages at specified interval:
+
+snippet: DelayedDeliveryProcessingInterval
+
+The poller picks and dispatches messages in batches of specified size:
```

Comment:
> Then the poller should be introduced earlier on and an explanation should be given about what it is and how it works.

We don't do that in any other place where we do polling, AFAIK. If we say instead "SQL Transport polls messages at intervals specified by..." then would that be acceptable?

> for non-optional params we should provide guidance on how to set them.

OK. Also came to my mind but all I could think about is giving an example, as we don't have any specific "perf tuning" techniques here I think.